emcc

```diff
@@ -1059,7 +1065,13 @@ try:
   # First, generate LLVM bitcode. For each input file, we get base.o with bitcode
   for i, input_file in input_files:
     file_ending = filename_type_ending(input_file)
-    if file_ending.endswith(SOURCE_ENDINGS):
+    if file_ending.endswith(JSOBJ_ENDINGS):
```

Comment:
How about putting this at the end of the if-elif chain, as it is the least important/clear to the typical reader.